docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -3,39 +3,86 @@ title: teleport-kube-agent Chart Reference
+| `Job`                 | The release name, suffixed by `-hook`                      | Used to delete legacy `Deployment`s in order to deploy a `StatefulSet` instead. Removed once the upgrade is complete.                                                                                                                                                                                   | If the `teleport-kube-agent` release contains a legacy `Deployment` resource.                                     |
+| `ConfigMap`           | The release name                                           | Contains the configuration for the Teleport pod.                                                                                                                                                                                                                                                        | Always.                                                                                                           |
+| `PodSecurityPolicy`   | The release name                                           | Enforces security requirements for pods deployed by `teleport-kube-agent`.                                                                                                                                                                                                                              | `podSecurityPolicy.enabled` is `true` and the Kubernetes cluster supports pod security policies.                  |
+| `Role`                | The release name, suffixed by `-psp`                       | Enforces security requirements for pods deployed by `teleport-kube-agent`.                                                                                                                                                                                                                              | `podSecurityPolicy.enabled` is `true` and the Kubernetes cluster supports pod security policies.                  |
+| `RoleBinding`         | The release name, suffixed by `-psp`                       | Enforces security requirements for pods deployed by `teleport-kube-agent`.                                                                                                                                                                                                                              | `podSecurityPolicy.enabled` is `true` and the Kubernetes cluster supports pod security policies.                  |
```

Comment:
I tried to apply `<br/>` tags to this table so the fist column wouldn't break, but couldn't find the magic placements to make it work. IMHO this is another example of why we should prioritize https://github.com/gravitational/docs/issues/126.